servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ClientSecurityConfigurator.java

```diff
@@ -51,64 +62,32 @@
      *
      * @param hostNameVerificationAlgorithm The algorithm to use when verifying the host name.
      * See <a href="https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#jssenames">
-     * Supported algorithm names</a>.
+     * Endpoint Identification Algorithm Name</a>.
      * @return {@code this}.
      * @see SSLParameters#setEndpointIdentificationAlgorithm(String)
      */
     ClientSecurityConfigurator hostnameVerificationAlgorithm(String hostNameVerificationAlgorithm);
 
     /**
-     * Determines what algorithm to use for hostname verification.
-     *
-     * @param hostNameVerificationAlgorithm The algorithm to use when verifying the host name.
-     * See <a href="https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#jssenames">
-     * Supported algorithm names</a>.
-     * @param hostNameVerificationHost the host name used to verify the
-     * <a href="https://tools.ietf.org/search/rfc2818#section-3.1">server identity</a>.
-     * @return {@code this}.
-     * @see SSLParameters#setEndpointIdentificationAlgorithm(String)
-     */
-    ClientSecurityConfigurator hostnameVerification(String hostNameVerificationAlgorithm,
```

Comment:
We should mark these methods as deprecated and keep them around for a few releases, because security configuration is used by many users.